### PR TITLE
Errors method

### PR DESCRIPTION
**Pull request by aniasobo**
Parser method that returns lovely error messages.

### CODE REVIEW

**Review by lucianmot — APPROVED** [review]
:snake: with a capital S

File: forest.py [hunk 1]
```diff
@@ -80,6 +81,10 @@ def get_token_by_key(self):
             if "STRING_CONTENT" in item:
                 return item["STRING_CONTENT"]
 
+    def return_error_message(self):
+        ERROR_MESSAGES = ["ERROR ʕノ•ᴥ•ʔノ ︵ =❱❯❭> =❱❯❭>", "Looks like you got lost in the Syntax Woods, Ranger ʕ·ᴥ·ʔ", "Forest does not know what this means ʅʕ•ᴥ•ʔʃ", "ERROR 🌲🌲🌲 ʕ·ᴥ·ʔ 🌲🌲🌲 YIKES", "٩ʕ•͡×•ʔ۶ This operation cant be completed", "Looks like you dont know Forest... but we dont know it either ⊂（´㉨｀*）⊃"]
```
Comment (josephtownshend) [inline forest.py:85]: love bears!

File: key_words.md [hunk 2]
```diff
@@ -90,8 +90,4 @@ BACKPACK:myvar^PACK_WITH^<<poop>>
 
 ### Error message ideas
 
-`ERROR ʕノ•ᴥ•ʔノ ︵ =❱❯❭> =❱❯❭>`  
-`Looks like you got lost in the Syntax Woods, Ranger ʕ·ᴥ·ʔ`  
-`Forest does not know what this means ʅʕ•ᴥ•ʔʃ `  
-`ERROR 🌲🌲🌲ʕ·ᴥ·ʔ🌲🌲🌲 YIKES`
```
Comment (josephtownshend) [inline key_words.md:96]: omg moar bearz

File: key_words.md [hunk 3]
```diff
@@ -90,8 +90,4 @@ BACKPACK:myvar^PACK_WITH^<<poop>>
 
 ### Error message ideas
 
-`ERROR ʕノ•ᴥ•ʔノ ︵ =❱❯❭> =❱❯❭>`  
-`Looks like you got lost in the Syntax Woods, Ranger ʕ·ᴥ·ʔ`  
-`Forest does not know what this means ʅʕ•ᴥ•ʔʃ `  
-`ERROR 🌲🌲🌲ʕ·ᴥ·ʔ🌲🌲🌲 YIKES`  
-`٩ʕ•͡×•ʔ۶ This operation cant be completed`
```
Comment (josephtownshend) [inline key_words.md:97]: bear FTW

**Review by josephtownshend — COMMENTED** [review]
brill

**Review by zepherine2006DEV — APPROVED** [review]
Nice! Merge!!

**Pull request merged**